pkg/storage/intent_resolver.go

```diff
@@ -302,26 +265,6 @@ func (ir *intentResolver) maybePushTransactions(
 // differently and would be better served by different entry points,
 // but combining them simplifies the plumbing necessary in Replica.
 func (ir *intentResolver) processIntentsAsync(r *Replica, intents []intentsWithArg) {
```

Comment:
Yeah, there isn't much point in not using the blocked goroutine. I'd like to do this in a follow-up PR, though. I'll file an issue.